### DISCUSSION TITLE
Test post

### DISCUSSION THREAD

**Discussion by ajasja** [RosettaCommons/rosetta#60] [Developing] [answered]
This is a test post with some 

```Python
test python code
```

That is not working.

**Comment by weitzner** [comment, marked as answer]
It is, you just don't have any python keywords in the snippet. See the slight difference below

```python
print("test python code")
```

**Comment by smlewis** [comment]
I interpreted that as "the python code is not working" [help me fix it] not "the web forum markup for python is not working"

(I am posting this entirely to play with the interface myself not meaningfully move the conversation forward)

**Comment by ajasja** [comment]
Thank you both for your help!!! @smlewis @weitzner
